composer.json

```diff
@@ -18,17 +18,20 @@
         "phpstan/phpstan": "^1.11"
     },
     "require": {
-        "php": "^8.2",
+        "php": "7.4.* | 8.0.* | 8.1.* | 8.2.* | 8.3.*",
+        "ext-json": "*",
+        "nyholm/psr7": "^1.8",
         "php-http/client-common": "^2.7",
         "php-http/discovery": "^1.19",
         "php-http/logger-plugin": "^1.3",
         "psr/http-client-implementation": "*",
-        "psr/log": "^3.0"
+        "psr/log": "*",
+        "symfony/http-client": "^5.4"
```

Comment:
How come symfony/http-client was added here?

Reply:
What version of composer did you use to create the file, just out of interest?

Reply:
I have moved the package to the  `require-dev` section

Reply:
> What version of composer did you use to create the file, just out of interest?

Probably the one in the php 8.2 dev container - v2.7.7

Reply:
The objective of the httplug abstraction is that we dont depend on a specific client implementation in the library and leave that up to the consumer - if we need an implementation for tests, there's a mock client we can use in require-dev: https://docs.php-http.org/en/latest/clients/mock-client.html

Reply:
Following on from conversation on slack, the `symfony/http-client` dependency is being left in the `require-dev` section for now